### ISSUE TITLE
how about remove the dust trade from bars?

### ISSUE THREAD

**Issue by gpmn** [bitshares/bitshares-ui#713]
the dust trade twist the graph heavily,  think about this suggestion, please

**Comment by svk31** [comment]
Closed as duplicate of a million other issues. It's not something that can be done in the GUI, waiting on the backend to fix it.

**Comment by abitmore** [comment]
This is duplicate of #200.
@svk31: any progress about https://github.com/bitshares/bitshares-core/pull/455 which is expected to fix this? If it's OK we will merge it.
